src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
```

Comment:
якщо назва змінної === ключу можна використати скороченний запис
```suggestion
    name,
    version,
    wheels,
```